deltachat-ios/Controller/Settings/AdvancedViewController.swift

```diff
@@ -56,17 +56,17 @@ internal final class AdvancedViewController: UITableViewController, ProgressAler
         return cell
     }()
 
-    private lazy var sendAutocryptMessageCell: ActionCell = {
-        let cell = ActionCell()
+    private lazy var sendAutocryptMessageCell: UITableViewCell = {
+        let cell = UITableViewCell(style: .value1, reuseIdentifier: nil)
```

Comment:
What's the reason vor `.value1` here? From the docs:

> A style for a cell with a label on the left side of the cell with left-aligned and black text; _on the right side is a label that has smaller blue text and is right-aligned._

Do we need that _smaller blue text_ or would `default` suffice, also for `manageKeysCell`?